docs/reference/commandline/inspect.md

```diff
@@ -63,7 +63,7 @@ $ docker inspect --format='{{.LogPath}}' $INSTANCE_ID
 ### Get an instance's image name
```

Comment:
Yes, I think we should be more descriptive here.

In this case, I actually think the example is correct, if `$INSTANCE_ID` refers to a _service_. `docker inspect` allows you to inspect various kind of objects (container, image, volume,
network, node, service, or task), so the examples should be clear _what_ kind of object is inspected.

Orthogonal to this PR; we may want to;

- explain where these placeholders come from (i.e., they follow the structure of the JSON that's shown if no `--format` option is set)
- link to the "formatting" https://docs.docker.com/engine/admin/formatting/ instead of the Go templating documentation (not sure why that document is in the "admin" section, we should change that; it can also use some improvement; it's not showing a lot of useful information